### PR TITLE
Fix format call

### PR DESCRIPTION
**Pull request by teymour-aldridge**
There was a missing 'v'

### CODE REVIEW

File: tabbycat/adjfeedback/views.py [hunk 1]
```diff
@@ -373,7 +373,7 @@ def get_tables(self):
                 'tooltip': _("Institution"),
             }, [team.institution.code if team.institution else TabbycatTableBuilder.BLANK_TEXT for team in tournament.team_set.all()])
 
-        adjudicators = tournament.adjudicator_set.all()
```
Comment (tienne-B) [inline tabbycat/adjfeedback/views.py:376]: What's the use of this line? (Why keep it at all?)

Comment (teymour-aldridge) [reply tabbycat/adjfeedback/views.py:376]: I don't know.

Comment (tienne-B) [reply tabbycat/adjfeedback/views.py:376]: I suggest you remove the line completely; the statement does not change the state, so does nothing useful by itself (not assigned).

**Comment by tienne-B** [comment]
Remove the f1da709 commit, as it's duplicated in your other PR (#1467) and is causing Travis to fail.

**Comment by tienne-B** [comment]
I'll merge this after #1463 is corrected, and f1da709 is removed. You can use [`git rebase`](https://git-scm.com/docs/git-rebase) for that. The command would be `git rebase -i 
f1da709^` then you delete the first line (in Vim, press `d` twice) and save (type `:x`).

**Comment by philipbelesky** [comment]
Hi @teymour-aldridge were you still interested in finish this off, or should we just directly commit the fix?

**Comment by teymour-aldridge** [comment]
I'm pretty busy at the moment; directly comitting would be good!

**Comment by tienne-B** [comment]
Removed the gitignore commit and merged the two commits that affected the same line.

**Pull request merged**